.github/workflows/smoke.yml

```diff
@@ -285,3 +285,29 @@ jobs:
       run: |
         export PERFORMANCE_TEST_MODE=${{ matrix.mode }}
         sudo -E make smoke-performance
+
+  failover-test:
+    runs-on: ubuntu-latest
+    needs: [contrib-build, nydus-build]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Download Nydus
+      uses: actions/download-artifact@v3
+      with:
+        name: nydus-artifact
+        path: target/release
+    - name: Download Nydusify
+      uses: actions/download-artifact@v3
+      with:
+        name: nydusify-artifact
+        path: contrib/nydusify/cmd
+    - name: Prepare Nydus Container Environment
+      run: |
+        sudo bash misc/failover/prepare.sh
+    - name: Failover Test
+      run: |
+        export SNAPSHOTTER_SYSTEM_SOCK=/run/containerd-nydus/system.sock
+        export SNAPSHOTTER=nydus
+        export FAILOVER_TEST_IMAGE=wordpress
```

Comment:
How about remove this and set up as the default value?

Reply:
Sure. I can check these envs and set the default values in code.

Reply:
Updated.